### ISSUE TITLE
not compatible A1111 last ben colab?

### ISSUE THREAD

**Issue by jvoxel** [zero01101/openOutpaint#122]
Internal Server Error? 

My guess is that is not yet implemented to run in colab notebooks but install flawlessly this thing looks great I wish I could use it inside colab too

**Comment by seijihariki** [comment]
Hi, actually, it seems someone was able to run this on colab, although as an extension: https://github.com/zero01101/openOutpaint-webUI-extension/issues/6 not sure how. @Lesteriax, did you use a publicly available notebook for stable diffusion webui?

**Comment by Lesteriax** [comment]
Hi @seijihariki, I'm actually using paperspace gradient notebook, not colab. The fix happened as follows:

I removed --gradio-debug (not sure where this would be found in other notebooks)
added --api in userbat
added --cors-allow-origins=https://1234567886.gradio.ap

I would basically launch the webui to get the link and then add it in userbat and refresh

**Comment by seijihariki** [comment]
Makes sense, thanks for your input! @jvoxel, would you be able to provide us the notebook you used for this error to crop up? Mostly to investigate the issue further.

**Comment by seijihariki** [comment]
A similar issue seems to have cropped up on another user of thelastben's colab.

As this seems to be extension related, will move this issue to the extension's repo and mark the other issue as duplicate.
